roles/neutronstar/tasks/main.yml

```
@@ -39,14 +52,19 @@
   lineinfile:
      dest: "{{ ngi_pipeline_conf }}/{{ bash_env_sthlm_script }}"
      line: >
-         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile standard,uppmax
-         -c {{ ngi_pipeline_conf }}/neutronstar_{{ item.site }}.config'
+         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile singularity \
```

```suggestion
         alias neutronstar='nextflow run {{ neutronstar_dest }}/main.nf -profile uppmax \
```

Why are we removing `-profile uppmax`? I think if we leave this then we can also simplify the config file from @remiolsen  a lot, to this:

```nextflow
process.container = '{{ neutronstar_container_path }}/{{ neutronstar_image }}'
params {
  busco_folder = '/sw/apps/bioinfo/BUSCO/v2_lineage_sets'
  busco_data = 'eukaryota_odb9'
}
```
I think `clusterOptions` and `multiqc_config ` etc should be in the general nextflow config that we use for other pipelines.

It would be best to keep the neutronstar config file as small as possible, only covering the stuff we really need to for _just_ the neutronstar pipeline. Then everything else is consistent across all pipelines 👍

---

Done!